app/models/like.rb

```diff
@@ -1,4 +1,11 @@
 class Like < ApplicationRecord
-  belongs_to :user
+  belongs_to :author, class_name: 'User'
   belongs_to :post
```

Comment:
As suggested by the previous reviewer, with the reasons stated above, please fix the same error in the `column_name` in the `likes` table, for your app to work 💪🤗

Your schema `t.bigint "user_id", null: false`
```ruby
create_table "likes", force: :cascade do |t|
  t.bigint "user_id", null: false
  t.bigint "post_id", null: false
  t.datetime "created_at", null: false
  t.datetime "updated_at", null: false
  t.index ["post_id"], name: "index_likes_on_post_id"
  t.index ["user_id"], name: "index_likes_on_user_id"
  end
```

expected `t.bigint "author_id", null: false`

```ruby
create_table "likes", force: :cascade do |t|
  t.bigint "author_id", null: false
  t.bigint "post_id", null: false
  t.datetime "created_at", null: false
  t.datetime "updated_at", null: false
  t.index ["author_id"], name: "index_likes_on_author_id"
  t.index ["post_id"], name: "index_likes_on_post_id"
end
```

Reply:
Hi @abdulhamiid 👋

# Changes
- Renamed columns 

![image](https://github.com/bohaz/BlogApp/assets/127757182/7d57c2c4-1a08-4922-bd72-80d90248e8f0)

- New wschema generated: Insted using `user_Id` now im using `author_id` both names are changed.

![image](https://github.com/bohaz/BlogApp/assets/127757182/561cb62f-d8f8-403c-bbab-6b3214ab5e4d)

- New comment created

![image](https://github.com/bohaz/BlogApp/assets/127757182/12394600-b321-444f-8a55-bd529c315811)